tensorflow/python/keras/benchmarks/perfzero_benchmark.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Copying is not good, but let's hold it here for now.

Reply:
Sounds great to me.